roles/pulp_devel/templates/alias.bashrc.j2

```diff
@@ -328,6 +326,12 @@ _pazurite_help="Switch to Azurite for Azure testing. For stopping it: pazurite s
 
 ptest () {
     comp="${1:-pulpcore}"
+
+    printf "Running tests using:\n\n"
+    echo "pytest -v -r sx --color=yes --suppress-no-test-exit-code --pyargs ${comp}.tests.functional -m parallel -n 8"
+    echo "pytest -v -r sx --color=yes --pyargs ${comp}.tests.functional -m \"not parallel\""
+    printf "\n\n"
+
```

Comment:
why duplicating it?

Reply:
This prints the lines but the other one runs them. I had heard that devs wanted to modify these to run a specific test, but all `ptest` would do is run them so you couldn't have a command easily to modify.

Reply:
oh! I didn't see the `echo`